### DISCUSSION TITLE
Getting error while running srsgnb

### DISCUSSION THREAD

**Discussion by Aniket621** [srsran/srsRAN_Project#235] [General Help]
Hi all, 

I am trying to build srsgnb and build is successfull but getting error while running srsgnb. As per gnb logs CU-CP and CU-UP Starting succcessfully and SCTP and NGAP interface showing up as per whireshark. 

gnb.logs 

![image](https://github.com/srsran/srsRAN_Project/assets/104416203/a77c3840-2f1f-4bcc-ac08-fc82d852e7de)

wireshark logs 
![image](https://github.com/srsran/srsRAN_Project/assets/104416203/f32c4a43-2acb-4f41-bc88-4088e52fee3a)

But getting error while running gnb and it's giving error  No radio resource available and unable to create radio session although zmq and czmq both are installed . 

![image](https://github.com/srsran/srsRAN_Project/assets/104416203/66938936-b4e1-44a9-b228-1f607ad00bff)

Kindly help above this error.

**Comment by robertfalkenberg** [comment]
Please share your config and the output of a fresh cmake run (i.e. empty build directory).

**Comment by Aniket621** [comment]
Hi Robest please find the attached cmake run output and configuration. 

AMF Conf:- 

![image](https://github.com/srsran/srsRAN_Project/assets/104416203/99892ec4-1282-47b1-bfb0-ca74f2f61f84)
gNodeb conf:- 

![image](https://github.com/srsran/srsRAN_Project/assets/104416203/2afe0bf1-2edc-4b36-a8b8-daa8b8a9bda3)
[cmake output.txt](https://github.com/srsran/srsRAN_Project/files/12812532/cmake.output.txt)

**Reply by robertfalkenberg** [reply]
Please run cmake with this extra option to enable ZeroMQ `-DENABLE_ZEROMQ=on`, then compile again.

**Reply by Aniket621** [reply]
Hi Robert Thanks for your feedback after enable ZeroMQ and compile again then srsRAN gNodeB is working. 
One question why we need to enable manually although it is already installed. 

![image](https://github.com/srsran/srsRAN_Project/assets/104416203/6a8b9e32-6b33-4743-8d97-936f7cc587f0)

**Comment by Aniket621** [comment]
Now Issue is resolve
